src/components/menu/Portal.tsx

```diff
@@ -0,0 +1,40 @@
+// (C) 2007-2018 GoodData Corporation
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
+
+export interface IPortalProps {
+    targetElement: Element;
+    children: React.ReactNode;
+}
+
+export default class Portal extends React.Component<IPortalProps> {
```

Comment:
Please try to find a name making it easier to infer what's the purpose and function of the component

Reply:
I thought about this but did not come up with anything better. 
I also think that since Portal is official React feature with relatively known name (like props) and the component is not doing anything other than creating React portal, then the name is descriptive enough. But if you come up with anything better I would be happy to rename this.

Reply:
Still... This portal is something else than the React portal so it deserves a specific name.
I'll think about it. `PortalRenderer`?

Reply:
yeah, for instance

Reply:
Renamed